scala-package/pom.xml

```diff
@@ -30,12 +30,13 @@
     <connection>scm:git:git@github.com:apache/incubator-mxnet.git</connection>
     <developerConnection>scm:git:git@github.com:apache/incubator-mxnet.git</developerConnection>
     <url>https://github.com/apache/incubator-mxnet</url>
-    <tag>mxnet-parent_2.11-1.2.0</tag>
+    <tag>HEAD</tag>
```

Comment:
Is this intended to point to the HEAD (master)?

Reply:
Yes, It has to create a new tag every time.

Reply:
I'm not very familiar with that setup, so please bare with me. What exactly is this supposed to do? I'd expect it to point to a certain branch, but this one would point to the master. What's the use-case here?

Reply:
Its just telling what tagName should be used when releasing. when HEAD it uses the name of the artifact defined in the pom file + version. We are not using this anyway, like I said on the email I have disabled tagging on the remote repo